Source/VFECore/Abilities/Ability.cs

```diff
@@ -89,29 +89,41 @@ public virtual bool IsEnabledForPawn(out string reason)
             (statFactor.value >= 0)
                 ? current * (this.pawn.GetStatValue(statFactor.stat) * statFactor.value)
                 : current / (this.pawn.GetStatValue(statFactor.stat) * Math.Abs(statFactor.value));
+        
+        private float CalculateStatOffsetForPawn(float current, StatModifier statOffset) =>
+            new[]{ToStringStyle.PercentZero, ToStringStyle.PercentOne, ToStringStyle.PercentTwo}.Contains(statOffset.stat.toStringStyle) ? 
+                current + (this.pawn.GetStatValue(statOffset.stat) - statOffset.stat.defaultBaseValue) * statOffset.value
+                : current + this.pawn.GetStatValue(statOffset.stat) * statOffset.value;
+        
+        private float CalculateModifiedStatForPawn(float current, IEnumerable<StatModifier> statFactors, IEnumerable<StatModifier> statOffsets) =>
+            statFactors.Aggregate(current, CalculateStatFactorForPawn) +
+            statOffsets.Aggregate(0f, CalculateStatOffsetForPawn);
```

Comment:
```suggestion
            statOffsets.Aggregate(statFactors.Aggregate(current, CalculateStatFactorForPawn), CalculateStatOffsetForPawn);
```
Vanilla does offsets then factors, so to be consistent we should too.

Reply:
> This seems unnecessarily confusing. Why can't you simply add the value of the stat to the current value?
There are different kinds of stats - some are expressed as a percentage where we would only want to offset by the amount above/below the base level. Some are straight numbers, where we would want to use it as a true offset.

For example:
Psychic sensitivity - statOffset * (currentSensitivity - baseSensitivity)
Movespeed (c/s) - statOffset * moveSpeed

Reply:
I think you responded to the wrong comment.